modules/storages/spec/common/storages/peripherals/nextcloud_connection_validator_spec.rb

```diff
@@ -230,6 +230,25 @@
     end
   end
 
+  context "when OpenID Connect is available" do
+    let(:storage) { create(:nextcloud_storage_configured, :oidc_enabled) }
+    let(:app_version) { Storages::SemanticVersion.parse("2.6.3") }
+    let(:capabilities_response) do
+      ServiceResult.success(result: Storages::NextcloudCapabilities.new(
+        app_enabled?: true,
+        app_version:,
+        group_folder_enabled?: false,
+        group_folder_version: nil
+      ))
+    end
+
+    it "errors if no Provider is available" do
```

Comment:
Should we add

```ruby
it "succeeds if a Provider is available"
```

?

(depending on taste in a nested context or immediately, like written above)

Reply:
Added!